person.rb

```diff
@@ -7,6 +9,7 @@ def initialize(age, name = 'unknown', parent_permision: true)
     @name = name
     @age = age
     @parent_permission = parent_permision
+    super()
```

Comment:
* Kindly fix this error I get when I try to test your decorator class, I think its because the correct name method is missing and its a [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Make%20sure%20that%20this%20class%20has%20a%20method%20correct_name%20implemented.%20It%20should%20simply%20return%20the%20name%20attribute.) to add it 
![image](https://user-images.githubusercontent.com/52497246/210833812-f0cc2f9c-1a89-451d-8bc1-eec15fa0e5f2.png)